src/kili/services/label_data_parsing/json_response.py

```diff
@@ -66,23 +113,75 @@ def __init__(
                 job_payload=job_response,
             )
 
-    def __getitem__(self, job_name: str) -> "job_response_module.JobPayload":
-        """Returns the JobPayload object corresponding to the job name."""
-        if job_name not in self._json_data:
-            raise JobNotExistingError(job_name)
-        return self._json_data[job_name]
-
-    @property
-    def jobs(self) -> "ParsedJobs":
-        """Returns self."""
-        return self
-
-    def to_dict(self) -> Dict:
+    def to_dict(self) -> Dict[str, Dict]:
         """Returns the parsed json response as a dict."""
         ret = {job_name: job_payload.to_dict() for job_name, job_payload in self._json_data.items()}
         ret = {k: v for k, v in ret.items() if v}  # remove empty json responses
         return ret
 
+
+def _is_video_job(project_info: Project, json_response: Dict) -> bool:
+    """Returns True if the json response is a video job, False otherwise."""
+    if "VIDEO" not in project_info["inputType"]:
+        return False
+
+    if not all(frame_id.isdigit() for frame_id in json_response.keys()):
+        return False
+
+    for frame_response in json_response.values():
+        if len(frame_response) == 0:
+            continue
+
+        for job_name in frame_response.keys():
+            if not (isinstance(job_name, str) and job_name in project_info["jsonInterface"]):
+                return False
+
+    return True
+
+
+class ParsedJobs(_ParsedOtherJobs, _ParsedVideoJobs):
+    """Class for label json response parsing."""
+
+    def __init__(
+        self,
+        project_info: Project,
+        json_response: Dict,
+        job_names_to_parse: Optional[List[str]] = None,
+    ) -> None:
+        # pylint: disable=line-too-long
+        """Class for label json response parsing.
+
+        This class will modify the input json_response.
+        If you want to keep the original json_response, use deepcopy.
+
+        Args:
+            project_info: Information about the project.
+            json_response: Value of the key "jsonResponse" of a label.
+            job_names_to_parse: List of job names to parse. By default, parse all the jobs that are not children.
+        """
+        self._is_video_job = _is_video_job(project_info, json_response)
```

Comment:
is_video_response

Reply:
since it both tests the project_info and json_response, the name is fine